pkg/chain/ethereum/block_counter.go

```diff
@@ -0,0 +1,8 @@
+package ethereum
+
+import "github.com/keep-network/keep-core/pkg/chain"
+
+// BlockCounter creates a BlockCounter that uses the block number in Ethereum.
+func (c *Chain) BlockCounter() (chain.BlockCounter, error) {
+	return c.blockCounter, nil
```

Comment:
What is the idea behind this function? THe block counter comes from `keep-common`, right? Maybe this function should be a part of `ethereum.go`? 🤔

Reply:
We could put in `ethereum.go` but wanted to keep it in the same way as others generic features and align with how interfaces are declared in `pkg/chain`. Also, if we want to merge `keep-common` one day, the block counter will live here.